@ndlib/gatsby-theme-marble/src/components/App/Pages/Portfolio/PortfolioBody/PortfolioEdit/EditItems/EditItem/EditItemForm/EditItemFormContent/index.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import TextField from 'components/App/FormElements/TextField'
+import TextArea from 'components/App/FormElements/TextArea'
+import MaterialButton from 'components/Internal/MaterialButton'
+import deleteIcon from 'assets/icons/svg/baseline-delete_forever-24px.svg'
+import closeIcon from 'assets/icons/svg/baseline-done-24px.svg'
+import { patchData, deleteData } from 'utils/api'
+import style from 'components/App/FormElements/style.module.css'
+
+export const EditItemFormContent = ({ item, closeFunc, deleteFunc, loginReducer }) => {
+  const [title, changeTitle] = useState(item.title)
+  const [annotation, changeAnnotation] = useState(item.annotation)
+  const [image, changeImage] = useState(item.image)
+  const [manifest, changeManifest] = useState(item.manifest)
+  const [link, changeLink] = useState(item.link)
+  const [patching, changePatching] = useState(false)
+  return (
+    <React.Fragment>
+      <div className={style.buttonGroup}>
+        <MaterialButton onClick={
+          (e) => {
+            e.preventDefault()
+            deleteItem(e, loginReducer, item.uuid, changePatching, deleteFunc, closeFunc)
+          }
+        }>
+          <img src={deleteIcon} alt='Delete' />
+        </MaterialButton>
+        <MaterialButton onClick={
+          // eslint-disable-next-line complexity
+          (e) => {
+            e.preventDefault()
+            const body = {
+              uuid: item.uuid,
+              title: title || null,
+              annotation: annotation || null,
+              image: image || null,
+              manifest: manifest || null,
+              link: link || null,
+            }
+            updateItem(e, loginReducer, body, changePatching, closeFunc)
```

Comment:
Why do you need the `|| null` for these properties? Are you expecting falsy values other than `null` and `undefined`, or does a function that uses this body object treat null and undefined differently?

Reply:
The api does not always behave correctly when an empty string is passed in, so it’s safer to send null.